web/modules/custom/joinup_search/joinup_search.behat.inc

```diff
@@ -205,6 +217,9 @@ class SearchSubContext extends DrupalSubContextBase implements DrupalSubContextI
    * @param string $facet
    *   The facet alias.
    *
+   * @throws \Exception
```

Comment:
You did good putting this exception here (though outside the scope), but since you did, the very next method, `\SearchSubContext::iClickAnInlineFacetItemLinkInRegion`, has exactly the same need for this documentation. You can add it there as well.